Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```diff
@@ -2542,19 +2542,19 @@ script:
       name: cve_id
       isArray: true
       required: false
-    - description: Severity of the CVE. Possible values CRITICAL, HIGH, MEDIUM, LOW, UNKNOWN, or NONE.
+    - description: Severity of the CVE. The possible values- CRITICAL, HIGH, MEDIUM, LOW, UNKNOWN, or NONE.
```

Comment:
The severity of the CVE. The possible values are: CRITICAL, HIGH, MEDIUM, LOW, UNKNOWN, or NONE.